eastlake/steps/coadd_nwgint.py

```diff
@@ -141,6 +141,12 @@ def _run(self, tilename, band, in_pyml, pyml, check_exists=False):
             jobs.append(joblib.delayed(run_and_check)(cmd, "CoaddNwgint", verbose=verbose))
 
         if len(jobs) > 0:
+            if "n_jobs" in self.config.keys():
+                n_jobs = self.config["n_jobs"]
+            else:
+                n_jobs = -1
+
+
```

Comment:
```suggestion
            n_jobs = self.config.get("n_jobs", -1)
```

Reply:
I'm happy with the code you wrote, but adding a comment just in case it helps in the future.

Reply:
That looks a lot cleaner, thanks!